server/command.go

```diff
@@ -60,6 +62,11 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, commandArgs *model.CommandArg
 	if len(args) == 0 || args[0] != "/webex" {
 		return p.help(commandArgs), nil
 	}
+
+	if !p.getConfiguration().IsValid() {
+		return p.responsef(commandArgs, "The Webex plugin has not been configured correctly: the sitename has not been set. Please contact your system administrator."), nil
```

Comment:
nit: what is `sitename`? Is it a term?

Reply:
@levb I think that's the term that the slack addon used, and what Cisco uses... But it's confusing, I agree. I called the `teamsite.webex.com` in the settings the "Webex Site Hostname". Is there a proper term for the hostname+domainname combination?